### DISCUSSION TITLE
Stealth settings vs. maximum protection, which one has better privacy?

### DISCUSSION THREAD

**Discussion by sith-on-mars** [kkapsner/CanvasBlocker#671] [Q&A]
I wonder between stealth settings and maximum protection, which preset offers better privacy protection? According to the EFF fingerprint test, stealth settings seemed to be better because my fingerprint wasn't unique, but with maximum protection I had a unique fingerprint. I'm not sure, however, the EFF fingerprint test is a good privacy measurement. Asking as a layperson here. Many thanks!

**Comment by kkapsner** [comment]
Having a unique fingerprint is not a problem when it changes all the time: https://canvasblocker.kkapsner.de/faq/#uniqueFingerprint

With maximum protection the web pages can (easier) detect that you are faking the fingerprint data. This itself can be used to fingerprint - but most of the time it is used to block access.

I would say "maxiumum protection" offers better protection (that's why I called it that). But everything is a tradeoff...
